lambdas/upload_ecr.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+set -e
+
+error() {
+    echo $@ 2>&1
+    exit 1
+}
+
+[[ $# == 2 ]] || error "Usage: $0 account_id image_name"
+
+account_id=$1
+image_name=$2
+
+regions=$(aws ec2 describe-regions --query "Regions[].{Name:RegionName}" --output text)
+
+for region in $regions
+do
+    if [[ $region == "eu-south-2" ]]
+    then
+        continue
+    fi
```

Comment:
I'll remove this as soon as we fix the repos.

Reply:
> as soon as we fix the repos

when we're planning to do that?

Reply:
I've manually added the repos and removed the hack